### DISCUSSION TITLE
繁體中文用詞請益

### DISCUSSION THREAD

**Discussion by cshung1994** [sparanoid/chinese-copywriting-guidelines#146] [Q&A]
#106 

修正繁體中文用詞，歡迎指教。

鍵接 `->` 超連結

指北 `->` 指南

**Comment by liyishuai** [comment]
普通話用戶看到“指北”時還以為是國語專屬用法。倘若國語和普通話都用“指南”，不妨一併改過來。

**Comment by WenryXu** [comment]
我理解这里的「指北」本身就是对「指南」一词的诙谐用法，所以无需更改。

**Comment by liyishuai** [comment]
鑒於本指南旨在高效精確地以中文交流，任何迷惑用法都应當避免。

**Comment by cshung1994** [comment]
本無「指北」一詞。
依據下文「統一中文文案、排版的相關用法，降低團隊成員之間的溝通成本，增強網站氣質。」，尚可將「指北」理解為「指南」屬刻意為之之用法。

### 補充詞源
- [指南](https://www.zdic.net/hans/%E6%8C%87%E5%8D%97)
- [超連結](http://terms.naer.edu.tw/detail/1678914/)

**Comment by liyishuai** [comment]
补充普通话用法：
- 指南：http://termonline.cn/list.htm?k=%E6%8C%87%E5%8D%97
- 超链接：http://termonline.cn/list.htm?k=hyperlink

**Comment by mamaruo** [comment]
一篇关于“指北”与“指南”的论述。
<img src="https://cdn.jsdelivr.net/gh/mamaruo/Image/img1/NorthOrSouth.png" width="40%">
[来源](http://www.cqvip.com/read/read.aspx?id=7813198)

**Comment by HumJ0218** [comment]
最开始看到这个项目，“指北”一次也让我一瞬间“找不着北”。我个人觉得，改成“指南”会相对通顺一些

**Comment by sgalal** [comment]
我也认为应该改成「指南」
